src/providers/auth0.js

```diff
@@ -10,7 +10,7 @@ class SalteAuthAuth0Provider {
    */
   static deauthorizeUrl(config) {
     return this.$utilities.createUrl(`${config.providerUrl}/v2/logout`, {
-      returnTo: config.redirectUrl,
+      returnTo: config.redirectUrl && config.redirectUrl.logoutUrl || config.redirectUrl,
```

Comment:
@jangolano 

This checks if redirectUrl contains a value called `logoutUrl`, if it doesn't it assumes `redirectUrl` is a `string`.